content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -142,13 +142,14 @@ the `istio-remote` helm chart's configuration values.
 
 | Helm Variable | Accepted Values | Default | Purpose of Value |
```

Comment:
@sdake Could this table be auto-generated?

Reply:
@geeknoid yes it can - although the python code needs a bit of change (it doesn't take any args, and would need to take an input file).  I'll modify the py code so it autogenerates for either multicluster-install or helm-install and add a followup PR to add autogeneration here.

Cheers
-steve